client/src/app/site/pages/meetings/pages/participants/export/participant-csv-export.service/participant-csv-export.service.ts

```diff
@@ -57,8 +58,41 @@ export class ParticipantCsvExportService {
         );
     }
 
-    public exportCsvExample(): void {
+    /**
+     * @param meeting
+     * @returns participants csv-example with added 'groups' value:
+     * - 2 custom group (not default or admin) names separated by comma by default
+     * - 1 custom group name if meeting has only 1 custom group
+     * - default group name if meeting has no custom groups
+     */
+    public addParticipantGroups(meeting: ViewMeeting): UserExport[] {
+        const customGroupNames = meeting.groups.filter(group => {
+            return !group.isAdminGroup && !group.isDefaultGroup;
+        });
+        let groupsToExport;
+
+        switch (customGroupNames.length) {
+            case 0:
+                groupsToExport = meeting.default_group.name;
+                break;
+            case 1:
+                groupsToExport = customGroupNames[0].name;
+                break;
+            default:
+                groupsToExport = customGroupNames
+                    .slice(0, 2)
+                    .map(group => group.name)
+                    .join(`, `);
```

Comment:
This can be simplified. 

`slice` will just use the length of the array when the end parameter exceeds its limits. For that reason `case 1` is not needed.